app/src/main/java/com/github/fribourgsdp/radio/game/GameActivity.kt

```diff
@@ -127,6 +137,12 @@ open class GameActivity : AppCompatActivity(), GameView, Timer.Listener {
     }
 
     override fun displayGuessInput() {
+        // Close the lyrics popup if already open
+        lyricsPopup?.let {
```

Comment:
Minor : I think codeclimate missed something : This block is repeated 4 times

Reply:
Done